templates/store/search.html

```diff
@@ -18,21 +18,6 @@
   <section id="main-content" class="p-strip--image is-shallow snapcraft-banner-background">
   <div class="row">
     <form action="/search" class="p-form p-form--inline p-form--search">
-      <div class="p-form__group p-form__group--no-grow">
```

Comment:
This this dropdown removed, is it now possible to search within a category? Or do we don't want this functionality anymore?

Reply:
It's still possible by URL hacking, but we decided to remove it as it's overly complicated and dead-end promoting on the UI